nih-sparc/sparc-app
# Find data filters #4a9p4j

# Description

The purpose of this PR is to change the filters for Find Data to be a dialog on all viewports. 

There is currently [a conversation about how the "selected filters" will function](https://app.clickup.com/t/4a9p4j) and it will be addressed in a future PR.

## Type of change


# How Has This Been Tested?

- Go to the Find Data page
- The table should be full width, and there should be a "Filters" button above the table.
- Click on the Filters button. Dialog should open.
- Select two of the filters. As you're selecting, filter tags should be appearing at the top.
- Click on one of the filter tags' X icon at the top to deselect it.
- The filters list should be updated
- Click "Apply"
- Dialog should be closed
- Click on the filters button. Dialog should be open showing the filters that you previously selected.
- Select all filters and close the dialog via the X button (*do not hit apply)
- Click on the filters button. Dialog should be open showing the filters that you selected before you hit the Apply button, not all filters that were previously selected.
- Click on the Clear All button
- Dialog should be closed
- Click on the filter button. Dialog should be open and should not have any filters checked.
- Click on the Organs tab
- The filters button should be disabled because there are no filters for organs

# Checklist:
